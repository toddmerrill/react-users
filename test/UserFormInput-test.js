import React from 'react';
import { UserFormInput } from '../src/components/Users.js';
import renderer from 'react-test-renderer';

it('UserFormInput renders correctly', () => {
    const user = {firstName: 'testFirst', lastName: 'testLast', age: 12345 }
    const tree = renderer.create(
        <UserFormInput user={user}/>
    ).toJSON();
    expect(tree).toMatchSnapshot();
});
