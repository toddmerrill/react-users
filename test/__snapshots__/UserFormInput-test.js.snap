// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserFormInput renders correctly 1`] = `
<div
  className="form-group"
>
  <label
    class="control-label"
    className="controlLabel"
    htmlFor="inputundefined"
  />
  <div
    className="controls"
  >
    <input
      autoFocus={undefined}
      className="form-control"
      id="inputundefined"
      name={undefined}
      onBlur={[Function]}
      onChange={[Function]}
      placeholder={undefined}
      tabIndex={undefined}
      type="text"
      value={undefined}
    />
  </div>
</div>
`;
